pkg/api/network.go

```diff
@@ -150,25 +150,15 @@ func (t *TitondAPI) GetK8sJobStatus(network *model.Network) (*appsv1.Deployment,
 }
 
 func (t *TitondAPI) createAccounts(namespace string) error {
-	sequencerKey, address := generateKey()
-	fmt.Printf("created sequencer account: %s\n", address)
-
-	proposerKey, address := generateKey()
-	fmt.Printf("created proposer account: %s\n", address)
-
-	relayerKey, address := generateKey()
-	fmt.Printf("created relayer account: %s\n", address)
-
-	signerKey, address := generateKey()
-	fmt.Printf("created block signer account: %s\n", address)
-
 	stringData := map[string]string{
-		"BATCH_SUBMITTER_SEQUENCER_PRIVATE_KEY": sequencerKey,
-		"BATCH_SUBMITTER_PROPOSER_PRIVATE_KEY":  proposerKey,
-		"MESSAGE_RELAYER__L1_WALLET":            relayerKey,
-		"BLOCK_SIGNER_KEY":                      signerKey,
```

Comment:
Thank you, Brave. Can you tell me why do you change this code? doesn't it work?

Reply:
I think if we generate accounts for using, we need another process to deposit ETH to it, then those accounts can make transactions. And i think it would be better if we keep the flow of simple-cluster with importing account from env or we receive from http request. So for the demo, i think we should keep the flow of simple-cluster

Reply:
It makes sense! Your deep idea always makes me realize.
@ohbyeongmin
I'm sorry, I missed demo. If you are okay, we can use brave's logic!